lib/skate_web/controllers/auth_controller.ex

```diff
@@ -71,4 +71,45 @@ defmodule SkateWeb.AuthController do
   def callback(%{assigns: %{ueberauth_failure: %{provider: :keycloak}}} = conn, _params) do
     send_resp(conn, :unauthorized, "unauthenticated")
   end
+
+  # https://github.com/mbta/arrow/blob/372c279e04866509f1e287e07844d61cc243850b/lib/arrow_web/controllers/auth_controller.ex#L57-L66
+  defp sign_out_url(auth) do
+    case UeberauthOidcc.initiate_logout_url(
+           auth,
+           %{
+             #  post_logout_redirect_uri: "https://www.mbta.com/"
```

Comment:
question: I didn't realize I left this in here commented out, I pulled this from https://github.com/mbta/arrow/pull/950/files#diff-94eca8259513c68e11b1b5d47d9c571dd7c0d8233a813ca95626507bffd65399R58-R60 but, any ideas on where we should redirect to after?

Reply:
I don't really know what the user flow looks like with this, so I'm testing in [`5b2d866` (#2380)](https://github.com/mbta/skate/pull/2380/commits/5b2d86647fc0d12af13a8f3d9d9d443f170f58a1)